packages/web/src/scss/components/TextField/index.html

```diff
@@ -205,6 +205,21 @@
     <button type="button" class="Button Button--primary Button--medium">Button</button>
   </div>
 
+  <div style="display: flex; align-items: start; gap: 1rem">
```

Comment:
@literat – I've added an example for inline TextField with message and submit button as you wished. Just to be sure: Are we using a similar layout somewhere in Jobs?
There has to be a small fix to prevent vertical expansion of the button because of flex.  There are more options how to fix it, but here I used `align-items: start;` and that's not possible with utility class in Jobs repository.

![image](https://user-images.githubusercontent.com/33354913/200024275-e60531c3-024b-4ce9-b20a-aee7293f195b.png)

Reply:
Anyway, if there will be label, this solution wouldn't be enough. But @crishpeen already implement a _small hack_ in Jobs repository. 🙂

Reply:
I am using the same solution on https://kukad.prod.internal.lmc/user where I am trying to test all components. Please check if your solution will also work there. Or I can create an issue for this problem and we can move on. Thanks.

Reply:
I don't have credentials. :/ Please check it yourself and then create an issue if necessary.

Reply:
We resolved this personally.